src/Servers/Kestrel/Transport.MsQuic/Internal/QuicEnums.cs

```diff
@@ -0,0 +1,265 @@
+// Copyright (c) .NET Foundation. All rights reserved.
+// Licensed under the Apache License, Version 2.0. See License.txt in the project root for license information.
+
+using System;
+
+namespace Microsoft.AspNetCore.Server.Kestrel.Transport.MsQuic.Internal
+{
+    /// <summary>
+    /// Represents the status of an operation by MSQuic
+    /// </summary>
+    public enum QUIC_STATUS : uint
```

Comment:
Let's `internal`ize everything we can, even though this is prototype. Better to start from safety :)